Makefile

```diff
@@ -68,7 +68,7 @@ GO_BUILD_TAG?=consul-build-go
 UI_BUILD_TAG?=consul-build-ui
 BUILD_CONTAINER_NAME?=consul-builder
 CONSUL_IMAGE_VERSION?=latest
-ENVOY_VERSION?='1.28.0'
+ENVOY_VERSION?='1.25.4'
```

Comment:
Guessing this is accidental diff from your other recent PR?

Reply:
thank you.  fixed this.